src/Mod/Part/App/FaceMaker.cpp

```diff
@@ -172,12 +264,12 @@ TYPESYSTEM_SOURCE(Part::FaceMakerSimple, Part::FaceMakerPublic)
 
 std::string Part::FaceMakerSimple::getUserFriendlyName() const
 {
-    return {QT_TRANSLATE_NOOP("Part_FaceMaker","Simple")};
+    return std::string(QT_TRANSLATE_NOOP("Part_FaceMaker","Simple"));
 }
 
 std::string Part::FaceMakerSimple::getBriefExplanation() const
 {
-    return {QT_TRANSLATE_NOOP("Part_FaceMaker","Makes separate plane face from every wire independently. No support for holes; wires can be on different planes.")};
+    return std::string(QT_TRANSLATE_NOOP("Part_FaceMaker","Makes separate plane face from every wire independently. No support for holes; wires can be on different planes."));
```

Comment:
@kaktusus can you look for this string in the translations? This looks suspect to me, I don't really expect a NOOP in this context. (@bgbsww nothing for you to worry about, we'll adjust in a later PR if this line is wrong).